README.md

```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+# Student Introduction: Elliot Karl
+### MACS 30500 HW Assignment #1
+### October 5, 2019
+
+####
+**About Me:** My name is Elliot Karl and I am a rising second year Masters of Public Policy Student (MPP) at the Harris School of Public Policy. I enrolled in *MACS 30500: Computing for Social Sciences* because I am hoping to:
+
+1. Build on the introductory knowledge I gained in using R at Harris
+2. Gain confidence in programming in R so that I can
+i) transform large budget data sets for future work in the local public sector which had become tedious (or impossible) in excel,
+ii) perform quick statistical summaries of datasets
+iii) perform this work in a way that would be reproducable and save me time in the long run.
+
+Prior to graduate school, I worked for years as a budget analyst at the City of Oakland in California. Here is a photo of me working on the [FY20-25 Capital Improvement Program](https://www.oaklandca.gov/documents/fy-2019-21-capital-improvement-program-cip):
+
+![Elliot Karl, circa June 2018](EK Headshot.jpg)
```

Comment:
I think Mint mentioned the space in the file name might be the issue here, but I also had some trouble with getting my photo to work. Eventually one of the TAs helped me realize I needed to create a local file (inside of the file where my markdown was stored so that I could link directly. Not sure if that would help here, but I think that inserting photos has a few caveats that weren't super clear from the syntax sheet.